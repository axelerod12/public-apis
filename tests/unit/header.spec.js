import { shallowMount } from '@vue/test-utils';
import FeldmHeader from '@/components/header/header.vue';

describe('FeldmHeader', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(FeldmHeader);
  });

  afterEach(() => {
    wrapper.destroy();
  });

  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot();
  });

  it('renders a nav', () => {
    const nav = wrapper.find('nav');
    expect(nav.exists()).toBe(true);
  });
});
