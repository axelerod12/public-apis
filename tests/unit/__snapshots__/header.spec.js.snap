// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeldmHeader renders correctly 1`] = `
<nav
  class="bg-blue-700 p-6 text-white"
>
  
  The official list of Public APIs

</nav>
`;
